### DISCUSSION TITLE
Switch Database Engine

### DISCUSSION THREAD

**Discussion by Sysadminfromhell** [modoboa/modoboa#2161] [Q&A]
Hi There, i forgot to change my Database Engine in the Installation process and now I have the problem that I run a WP Website with the Modoboa and MySql Engine for WP and the postgresql from Modoboa. Is there anyway to Migrate or replace the engine?

**Comment by tonioo** [comment]
@Sysadminfromhell There is no easy way to switch database engine. If your setup is recent, I suggest you run the install once again with mysql enabled (change the value in the config file). Don't forget to backup you mysql databases first! Once your done, just remove postgresql.

**Reply by Sysadminfromhell** [reply]
Okay, how can I backup the postressql and import it to MySQL ?

**Reply by tonioo** [reply]
Maybe try to dump the SQL statements of the modoboa database (using pg_dump), convert the result to MySQL compatible statements and finally import them.
